CMakeLists.txt

```diff
@@ -160,6 +162,14 @@ if(HAVE_UPDWTMPX)
     )
 endif()
 
+if(HAVE_UTEMPTER)
+    target_compile_definitions(${QTERMWIDGET_LIBRARY_NAME}
+        PRIVATE
+            "HAVE_UTEMPTER"
+    )
+    target_link_libraries(${QTERMWIDGET_LIBRARY_NAME} ulog)
```

Comment:
Linking to ulog is not so intuitive for an application using utempter.h. Is there a documentation for this?